example_client/multi_inputs.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python
+#
+# Copyright 2019, Intel Corporation
```

Comment:
Please update licence header. 
It isn't a full Apache licence header

Reply:
Done